l10n_it_ricevute_bancarie/models/account/account.py

```diff
@@ -137,4 +142,99 @@ def _get_invoice_by_move_line(self, cr, uid, ids, context=None):
                     'unsolved_invoice_ids', 'reconcile_id'], 10),
                 }
             ),
+
         }
+
+    def _get_first_date_due(self):
+        pterm = self.env['account.payment.term'].browse(self.payment_term.id)
+        pterm_list = pterm.compute(value=1, date_ref=self.date_invoice)[0]
+        return min(line[0] for line in pterm_list)
+
+    def month_check(self, invoice_date_due, all_date_due):
+        """
+
+        :param invoice_date_due: first date due of invoice
+        :param all_date_due: list of date of dues for partner
+        :return: True if month of invoice_date_due is in a list of all_date_due
+        """
+        check = False
+        for d in all_date_due:
+            if invoice_date_due[:7] == d[:7]:
+                check = True
```

Comment:
Can you return immediately True to first true condition and return False as last possibility?